### DISCUSSION TITLE
How to change the color of inline code block?

### DISCUSSION THREAD

**Discussion by seanSeungeonKim** [mmistakes/minimal-mistakes#2808] [Q&A] [answered]
I think the default is not that noticeable compared to plain text so I want to customize it to something like Slack style
<img width="228" alt="Screen Shot 2021-01-27 at 12 43 48 AM" src="https://user-images.githubusercontent.com/58868020/105948313-ba99aa80-6038-11eb-9efe-9b063d49d24d.png">
<img width="89" alt="Screen Shot 2021-01-27 at 12 44 07 AM" src="https://user-images.githubusercontent.com/58868020/105948350-c5ecd600-6038-11eb-82b9-221068a10827.png">
which scss file should I touch on?

**Comment by PhilKeeble** [comment, marked as answer]
You could go into the css for the skin you are using and add something like:

```
code {
    color: orange;
}
```

Then you could just customise in there and it will change for all code blocks. For span code blocks like above, you can keep the normal highlighting by specifying the language after the three backticks
